app/scripts/app.js

```diff
@@ -42,6 +47,23 @@ angular.module('yourStlCourts').config(function($stateProvider, $urlRouterProvid
         }
       }
     })
+    .state('courtSearchInfo', {
+      url: '/courts/{courtId}',
+      templateUrl: 'views/courtSearchInfo.html',
+      controller: 'CourtSearchInfoCtrl as ctrl',
+      params: {
+        court: {value: undefined}
```

Comment:
params would include courtId, not court. Having this params block here doesn't do anything